### PR TITLE
fix: select text in validation console

### PR DESCRIPTION
**Pull request by roshan-gh**

### CODE REVIEW

File: packages/otelbin/src/components/monaco-editor/ResizeBar.tsx
Comment (roshan-gh) [inline packages/otelbin/src/components/monaco-editor/ResizeBar.tsx:1]: @bripkens selecting text in validation console fixed

File: packages/otelbin/src/components/monaco-editor/ResizeBar.tsx [hunk 1]
```diff
@@ -22,7 +22,6 @@ export function ResizeBar({ onWidthChange }: ResizeBarProps) {
 		};
 
 		const onMouseMove = (e: MouseEvent) => {
-			e.preventDefault();
```
Comment (bripkens) [inline packages/otelbin/src/components/monaco-editor/ResizeBar.tsx:25]: **suggestion:** I believe we should still call `e.preventDefault()` when `state.current.dragging`. This ensures that text selection doesn't happen when resizing the editor.

Comment (bripkens) [reply packages/otelbin/src/components/monaco-editor/ResizeBar.tsx:25]: yep, seems to only occur in the error panel. Which is okay.

**Review by bripkens — APPROVED** [review]

File: packages/otelbin/src/components/monaco-editor/ResizeBar.tsx (hunk 1 above)
Comment (roshan-gh) [reply packages/otelbin/src/components/monaco-editor/ResizeBar.tsx:25]: @bripkens hmm it's strange, I tested in Chrome, and there was no extra selecting text in anywhere.

**Pull request merged**